### PR TITLE
Bind PackageMap fully in Python and update RBT python constructor

### LINKED ISSUE #5520: Cannot construct Atlas model using pydrake bindings
```python
import pydrake.rbtree as rbt
robot = rbt.RigidBodyTree("/home/rdeits/locomotion/drake-distro/drake/examples/Atlas/urdf/atlas_minimal_contact.urdf")
```

gives:

```
RuntimeError: /home/rdeits/locomotion/explorations/drake-mpc/build/drake-prefix/src/drake/drake/multibody/parsers/urdf_parser.cc: ParseGeometry: ERROR: Mesh file name could not be resolved from the provided uri "package://Atlas/urdf/meshes/r_clav.obj".
```

It looks like that constructor needs to be updated to use the package path. I'll look into it.

### PR DESCRIPTION
**Pull request by rdeits**
Fixes #5520

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/5523)

### CODE REVIEW

**Comment by rdeits** [comment]
+@jwnimmer-tri for platform review

---

Review status: 0 of 6 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/5523#-:-KfNQGA3PVNSFthuh3JU:b-trzr01)*

File: drake/bindings/python/pydrake/rbtree.py [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 from ._pydrake_rbtree import *
 
+# TODO: Remove this direct import and ask users to import
+# pydrake.parsers.PackageMap instead
+from .parsers import PackageMap
```
Comment (patmarion) [inline drake/bindings/python/pydrake/rbtree.py]: thanks for making it backward compatible.  im happy to update director code if/when you want to remove this.

**Comment by jwnimmer-tri** [comment]
:lgtm:

It would be good to have a second reviewer give it a read, but if you can't find any volunteers I think its okay to merge with a single review.

---

Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/bindings/python/pydrake/test/testPackageMap.py, line 47 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/5523#-KfNRvUGbQ6a6E8lYPzU:-KfNRvUGbQ6a6E8lYPzV:b9o829k) ([raw file](https://github.com/robotlocomotion/drake/blob/a0b09a210ac03a3da6515d5a49aaad983dfd9537/drake/bindings/python/pydrake/test/testPackageMap.py#L47)):*
> ```Python
> 
> if __name__ == '__main__':
>     unittest.main()
> ```

BTW Forgive me for not yet knowing Drake's pyguide by heart -- but in past projects' python coding standards we had a rule that files should have a dunder-name-dunder-main function exactly when they start with a shebang line such as `#/usr/bin/env python2`.  Is that currently a rule in Drake's python guide?  Should these files have shebang lines added?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/5523#-:-KfNSYKOH1ixC-Tk3OT3:b-xetx9i)*

**Comment by rdeits** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[drake/bindings/python/pydrake/test/testPackageMap.py, line 47 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/5523#-KfNRvUGbQ6a6E8lYPzU:-KfNU8mTJwiOPULZX22n:bjr1on9) ([raw file](https://github.com/robotlocomotion/drake/blob/a0b09a210ac03a3da6515d5a49aaad983dfd9537/drake/bindings/python/pydrake/test/testPackageMap.py#L47)):*
<details><summary><i>Previously, jwnimmer-tri (Jeremy Nimmer) wrote…</i></summary><blockquote>

BTW Forgive me for not yet knowing Drake's pyguide by heart -- but in past projects' python coding standards we had a rule that files should have a dunder-name-dunder-main function exactly when they start with a shebang line such as `#/usr/bin/env python2`.  Is that currently a rule in Drake's python guide?  Should these files have shebang lines added?
</blockquote></details>

I don't think that's a convention we're using in Drake. I added it here so that you can run an individual test with `python -m pydrake.test.testQuadraticProgram` (which runs the test by loading it as a module, rather than by sourcing it as a script). 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/5523)*

**Comment by rdeits** [comment]
+@patmarion could you be the other reviewer?

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/5523#-:-KfNUrYPZLE0brUbeLmH:b-gwcoxh)*

**Comment by patmarion** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[drake/bindings/pybind11/CMakeLists.txt, line 58 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/5523#-KfNVN2_8_p_cGybnvMP:-KfNVN2_8_p_cGybnvMQ:bo9twj7) ([raw file](https://github.com/robotlocomotion/drake/blob/a0b09a210ac03a3da6515d5a49aaad983dfd9537/drake/bindings/pybind11/CMakeLists.txt#L58)):*
> ```CMake
> set_target_properties(_pydrake_parsers PROPERTIES OUTPUT_NAME "parsers")
> target_compile_options(_pydrake_parsers PRIVATE "-fvisibility=default")
> install(TARGETS _pydrake_parsers DESTINATION ${PYDRAKE_INSTALL_DIR})
> ```

BTW, in the future i think a macro or function could help reduce the repeated cmake calls

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/5523#-:-KfNVqszvVxXHrIo5YC_:bnfp4nl)*

**Comment by rdeits** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[drake/bindings/pybind11/CMakeLists.txt, line 58 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/5523#-KfNVN2_8_p_cGybnvMP:-KfNWAn1i2qs16ZVAifd:b-fhb2u4) ([raw file](https://github.com/robotlocomotion/drake/blob/a0b09a210ac03a3da6515d5a49aaad983dfd9537/drake/bindings/pybind11/CMakeLists.txt#L58)):*
<details><summary><i>Previously, patmarion (Pat Marion) wrote…</i></summary><blockquote>

BTW, in the future i think a macro or function could help reduce the repeated cmake calls
</blockquote></details>

True. But if this is all moving to Bazel soon, it's probably not worth the effort right now. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/5523)*

**Pull request merged**